conda/models/records.py

```diff
@@ -418,6 +423,11 @@ def record_id(self):
             self.channel.name, self.subdir, self.name, self.version, self.build
         )
 
+    @property
+    @lru_cache(maxsize=None)
+    def metadata(self) -> Metadata:
+        return Metadata()
```

Comment:
We shouldn't wrap methods with `lru_cache` since the garbage collector can't clean it up (https://rednafi.com/python/lru_cache_on_methods/) or am I missing a reason here for this somehow?

```suggestion
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.metadata = lru_cache(maxsize=None)(self._metadata)

    def _metadata(self) -> Metadata:
        return Metadata()
```

Reply:
Or would this suffice?

```suggestion
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.metadata = Metadata()
```

Reply:
see https://github.com/conda/conda/pull/13053/commits/ce45724620ab7f09605e909e271b8e2ec1294242